.travis.yml

```diff
@@ -1,17 +1,17 @@
 language: python
 python:
-  - "2.7"
-  - "3.4"
+  - 3.6
 install:
-    - pip install -r requirements.txt
-    - pip install braintree coveralls mock mysqlclient
+  - pip install -r requirements.txt
+  - pip install braintree coveralls mock 
 env:
-  - DB=sqlite
-  - DB=mysql
   - DB=postgres
```

Comment:
You also removed sqlite and mysql from here @Jak-Sidious  ..why?

Reply:
As the team is all using Postgres, I didn't think it was necessary to have the additional environments. However, I shall make the additions accordingly